srml/council/src/seats.rs

```diff
@@ -446,24 +454,24 @@ impl<T: Trait> Module<T> {
 		Ok(())
 	}
 
-	/// Remove a voter from the system. Trusts that Self::voters()[index] != voter.
+	/// Remove a voter from the system. Trusts that `Self::voters()[index] != voter`.
 	fn remove_voter(voter: &T::AccountId, index: usize, mut voters: Vec<T::AccountId>) {
 		<Voters<T>>::put({ voters.swap_remove(index); voters });
```

Comment:
instead of writing the comment `Trusts that Self::voters()[index] != voter`, what about the following?

```suggestion
                assert_ne!(Self::voters()[index] != voter);
		<Voters<T>>::put({ voters.swap_remove(index); voters });
```

Reply:
I'm definitely not the Rust expert, but I don't think that will compile because there is no return value for the function. An `assert_{x}!` macro requires that you can return an `Err`.

Reply:
yes i meant to change that to a comma. updated now.

Reply:
No assertions (or any panics) are allowed inside the runtime. If needed, the code and the comments should act as proof that some criteria always persists.